### PR TITLE
Output a summary when there are newer version, but they cannot resolv…

### PR DESCRIPTION
**Pull request by sigurdm**
…e together

### CODE REVIEW

**Comment by sigurdm** [comment]
cc @mit-mit

File: lib/src/command/outdated.dart [hunk 1]
```diff
@@ -322,6 +322,11 @@ Future<void> _outputHuman(List<_PackageDetails> rows,
           'To update these dependencies, edit pubspec.yaml.');
     }
   }
+
+  if (notAtResolvable == 0 && upgradable == 0 && rows.isNotEmpty) {
+    log.message(
+        '\nDependencies are all on the latest mutually compatible versions');
```
Comment (jonasfj) [inline lib/src/command/outdated.dart]: I think we should add a period.

Or maybe write something like:
```
'\nNewer versions is available for $N dependenc(y|ies), but these are
 not mutually compatible with your other dependencies.'
```

OR
```
'\nDependencies are all on the latest mutually compatible versions'
'\nNewer versions, while available, is not mutually compatible'
```

File: lib/src/command/outdated.dart [hunk 2]
```diff
@@ -325,7 +325,8 @@ Future<void> _outputHuman(List<_PackageDetails> rows,
 
   if (notAtResolvable == 0 && upgradable == 0 && rows.isNotEmpty) {
     log.message(
-        '\nDependencies are all on the latest mutually compatible versions');
+        '\nDependencies are all on the latest mutually compatible versions.'
```
Comment (mit-mit) [inline lib/src/command/outdated.dart]: Maybe s/Dependencies are all on the latest mutually compatible versions/Dependencies are all on the latest resolvable versions/ given we have the phrase 'mutually compatible below'?

Comment (sigurdm) [reply lib/src/command/outdated.dart]: Done

**Pull request merged**